test/onnx/test_utility_funs.py

```diff
@@ -370,6 +370,49 @@ def forward(self, x):
             assert node.kind() != "onnx::Mul"
         assert len(list(graph.nodes())) == 1
 
+    # TODO : enable when constant folding is enabled for opset 12
+    @skipIfUnsupportedOpsetVersion([12])
+    def test_constant_fold_add(self):
+        class Module(torch.nn.Module):
+            def __init__(self, ):
+                super(Module, self).__init__()
+                self.register_buffer("weight", torch.ones(5))
+
+            def forward(self, x):
+                add = self.weight + torch.tensor([1, 2, 3, 4, 5])
+                return add - x
+
+        x = torch.randn(2, 5)
+        _set_opset_version(self.opset_version)
+        _set_operator_export_type(OperatorExportTypes.ONNX)
+        graph, _, __ = utils._model_to_graph(Module(), (x, ), do_constant_folding=True,
+                                             operator_export_type=OperatorExportTypes.ONNX)
+        for node in graph.nodes():
+            assert node.kind() != "onnx::Add"
+        assert len(list(graph.nodes())) == 1
```

Comment:
Should we also add a value check to verify the correctness of the constant folding behavior?

Reply:
Done